test/README.md

```diff
@@ -0,0 +1,22 @@
+# execute test files
+
+## test runs
+```npm run perf```
+```npm run test```
+
+## Cache files
+```cd test```
+
+```npx ts-node --esm src/cache.ts```
+
+## IndexedDB
+
+### test script
+
+1. uncomment fake DB linesin the `src/index.ts` library
```

Comment:
We'll have a proper testing implemented once I resolve the issue with the jest. and hopefully get rid of this kinds of tricks completely including the whole test folder.